lib/routes/sycltech/namespace.ts

```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
```

Comment:
Please use [namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) `sycl` instead of `sycltech`.